### DISCUSSION TITLE
Getting too many "Update Dependency Injection PRs".

### DISCUSSION THREAD

**Discussion by Sardor-M** [renovatebot/renovate#22036] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

I have been getting too much PR creation from the Renovate Bot from different kinds of dependency update injection. 

I was looking for a solution to control updating them only once a week. 

I would appreciate your help and ideas regarding on how to solve this issue. 

Thank you. 

### What you tried so far.

For the record, i get too many update dependency Pull Requests at random time of the Week which is annoying to see. 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by HonkingGoose** [comment, marked as answer]
Please read [our noise reduction docs](https://docs.renovatebot.com/noise-reduction/), it covers:

- Grouping multiple packages into one update
- Scheduling when Renovate creates PRs
- Automerging dependencies (make sure you have good enough test coverage!)
- Lock file considerations

Also read up on ["the Dependency Dashboard Approval"-workflow](https://docs.renovatebot.com/key-concepts/dashboard/#dependency-dashboard-approval-workflow). You can configure Renovate so it "asks for approval" before creating a PR for certain types of updates, or for specific packages.

**Reply by Sardor-M** [reply]
Thank you for your answer, I figured it out how to enable it through the documentation.
